README.md

```diff
@@ -10,80 +10,78 @@ A framework for building Mobile cross-platform UI.
 | iOS | [![Pod version](https://badge.fury.io/co/WeexSDK.svg)](https://cocoapods.org/pods/WeexSDK) [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage) |
 | [Mobile Web](https://github.com/weexteam/weex-vue-render) | [![npm version](https://badge.fury.io/js/weex-html5.svg)](https://www.npmjs.com/package/weex-vue-render) |
 
-Support Android 4.1 (API 16), iOS 8.0+ and WebKit 534.30+.
+Supports Android 4.1 (API 16), iOS 8.0+ and WebKit 534.30+.
 
 ## For Windows
 
 Please ***INSTALL [Git for Windows](https://git-scm.com/download/win)*** and run all the following commands in git-bash.
 
 ## Meet Weex
 
-* Install [Weex Playground App](https://weex.apache.org/tools/playground.html) to see examples we already written.
-* If you want to write a demo, install [weex-toolkit](https://www.npmjs.com/package/weex-toolkit) in [Node.js](http://nodejs.org/) 4.0+ and
-* Run `weex init` to generate & start a simple project in an empty folder.
-* Follow the instructions in the project README.
+* Install [Weex Playground App](https://weex.apache.org/tools/playground.html) to see existing examples.
+* Create your own examples by install [weex-toolkit](https://www.npmjs.com/package/weex-toolkit) with [Node.js](http://nodejs.org/) 6.0+.
```

Comment:
Since the `engine` property in the [`package.json`](https://github.com/apache/incubator-weex/blob/0.18.0/package.json#L34-L36) is `>=8`, we can declare the version of node as `8.0+`.

Reply:
@Hanks10100 Thanks for pointing it out. I will update it.

I thought originally, it was referencing the Node.js requirement of `weex-toolkit` which is currently `>=6.0` in master, not the entire `incubator-weex`.

https://github.com/weexteam/weex-toolkit/blob/master/package.json#L13